mammalia.csl

```diff
@@ -52,14 +53,43 @@
     </names>
   </macro>
   <macro name="title">
-    <text variable="title"/>
+    <text variable="title" font-style="italic"/>
```

Comment:
That's not right, is it? Looks like should be italics just for books & similar, not for journals & chapters?